backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -557,9 +571,9 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     * Returns true if the status represents a success.
     *
     * @param runStatus The run status.
-    * @return True if the job is a success.
+    * @return True if the job is a done.
```

Comment:
`True if the job is a done.` should be `True if the job is done.`

Reply:
or `True if the job is a done deal.`